spec/models/shelter_spec.rb

```diff
@@ -2,10 +2,9 @@
 
 describe Shelter, type: :model do
   describe "validations" do
-    it {should validate_presence_of(:name)}
-    it {should validate_presence_of(:address)}
-    it {should validate_presence_of(:city)}
-    it {should validate_presence_of(:state)}
-    it {should validate_presence_of(:zip)}
```

Comment:
I changed this because I feel the pet name is the only thing completely necessary, but will reevaluate down the line